tpgtools/overrides/compute/beta/firewall_policy_rule.yaml

```diff
@@ -1,4 +1,3 @@
 - type: DIFF_SUPPRESS_FUNC
-  field: target_resources
```

Comment:
Removing this will result in the diff suppress funtion `CompareSelfLinkOrResourceName` being removed from the field `target_resources` for providers (https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-7940-old..auto-pr-7940). Is this removal intentional?

Reply:
Yes, I've removed that, I'm using this file as example: https://github.com/GoogleCloudPlatform/magic-modules/blob/main/tpgtools/overrides/compute/network_firewall_policy_rule.yaml
And they don't use this field there. But it makes more sense to keep that in the code. Will fix that.